python/ray/serve/controller.py

```diff
@@ -317,6 +334,8 @@ async def run_control_loop(self) -> None:
             except Exception:
                 logger.exception("Exception updating application state.")
 
+            self._update_active_nodes()
```

Comment:
maybe put this just above `http_state.update` given they're very related and it'll keep them consistent